Project.toml

```diff
@@ -3,11 +3,22 @@ uuid = "a14a6c0f-317a-4379-b4a0-4ba4e1e9b5ed"
 authors = ["Johnny Chen <johnnychen94@hotmail.com>"]
 version = "0.1.0"
 
+[deps]
+FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
+ImageFiltering = "6a3955dd-da59-5b1f-98d4-e7296123deb5"
+ImageIO = "82e4d734-157c-48bb-816b-45c225c6df19"
```

Comment:
`ImageIO` is also a test-dependency, ImageIO is only available for Julia >= v1.3, so we need `ImageMagick` as well.

Reply:
Yes, I have found my mistake. I hadn't found `using ImageMagick` in any code. So I considered that this package is no use and didn't add this pkg. But the truth was I'm wrong. I'm going to add `ImageMagick` to the test-dependency.

Reply:
I'd like to know whether I need to delete `ImageIO` in the dependency?

Reply:
> I hadn't found using ImageMagick in any code.

This is a FileIO design (where `load`/`save` is defined) that IO is separated into two parts: 1) the frontend API `load` and `save`, and 2) the various backends for different formats, you can't require users to install all available backends (there're >30 outside) so FileIO lazily calls for example `using ImageMagick; ImageMagick.load(...)` somewhere.

Basically, to load an image, we need to install the frontend `FileIO` and then some io backend `ImageIO` and/or `ImageMagick` to make it work. [TestImages also uses `FileIO`](https://github.com/JuliaImages/TestImages.jl/blob/master/Project.toml)

> I'd like to know whether I need to delete ImageIO in the dependency?

We can still keep `ImageIO` as it's more lightweight and faster than `ImageMagick`. The JuliaImages stack currently uses both by default https://github.com/JuliaImages/Images.jl/pull/948